### DISCUSSION TITLE
arduino uno is not connecting to UGS

### DISCUSSION THREAD

**Discussion by Yashk09** [winder/Universal-G-Code-Sender#2330] [Help]
i am having "*** Connecting to jserialcomm://COM10:115200
Grbl 0.9j ['$' for help]

?
<Idle,WPos:2147
***connection closed ***"
this issue while connecting to UGS
please help me to solve this issue .

and i am using Grbl 0.9j and nighty build UGS 
and arduino uno R3 board.

**Comment by breiler** [comment]
It looks like you didn't get a complete response from the controller (it is chopped off at the end):
```
?
<Idle,WPos:2147
***connection closed ***"
```

I am guessing that either the USB cable or the Arduino is not of sufficient quality. First make sure you have a properly shielded USB cable. If you are using a cheap Chinese arduino clone, consider in investing in a genuine Arduino.

**Comment by Yashk09** [comment]
Yes I am using clone Arduino but 1st time this works perfectly fine but now
it's not connecting

**Reply by noweare1** [reply]
Did you find a solution for this. I am getting the same thing happening. Changed arduino board to a genuine board and tried a different cable with same results. The funny thing is that it was connecting fine and at some time stopped. Real head scratcher this one.

**Reply by breiler** [reply]
Can you activate verbose logging in the console (right click in the console and select it in the menu) and connect again. 

Paste the console output here

**Reply by noweare1** [reply]
This is with verbose on. 
*** Connecting to jserialcomm://ttyACM0:57600
Grbl Eh Servo 0.9j ['$' for help]
Grbl Eh Servo 0.9j ['$' for help]
*** Fetching device status
>>> ?
<Idle,MPos:0.000,0
Grbl Eh Servo 0.9j ['$' for help]
Grbl Eh Servo 0.9j ['$' for help]
*** Fetching device status (2 of 3)...
>>> ?
<Idle,MPos:0.000,0
Grbl Eh Servo 0.9j ['$' for help]
Grbl Eh Servo 0.9j ['$' for help]
*** Fetching device status (3 of 3)...
>>> ?
<Idle,MPos:0.000,0
Grbl Eh Servo 0.9j ['$' for help]
Grbl Eh Servo 0.9j ['$' for help]
**** Connection closed ****

**Reply by breiler** [reply]
Yeah, the response from the controller is corrupted and is likely due to a bad USB cable or maybe a bad Arduino. Make sure that you are using a properly shielded USB cable without an extensions or USB hubs.

**Comment by noweare1** [comment]
I was able to connect. I wish I could tell you why but I do not know what changed. Ugh. I hope this isn't temporary. Thank you for responding.
